metabase/metabase
# Fix global error page

Reverts https://github.com/metabase/metabase/pull/25554

The PR above doesn't work in production since (any) rendering error would result in a complete blank app. This means:
- Error counting logic in the PR above doesn't make sense in production
- `AppErrorCard` is never displayed in production

This PR:
- Reverts the error counting logic
- Removes `AppErrorCard` 
- Handles the rendering error in the same way as other errors in the app are handled. In this case a generic error screen would be displayed

How to test:
- Open `components/HomePage.tsx` and add `throw new Error();` before returning anything
- Build the app `./bin/build`
- Run `java -jar ./target/uberjar/metabase.jar`
- Make sure that when opening the homepage you get the error screen instead of the blank screen

<img width="1006" alt="Screenshot 2022-10-03 at 14 40 26" src="https://user-images.githubusercontent.com/8542534/193568512-2cf543e9-0132-4080-9f46-33ad395c3919.png">
